config/locales/views/transactions/en.yml

```
@@ -48,9 +68,12 @@ en:
       potential_duplicate_description: This pending transaction may be the same as the posted transaction below. If so, merge them to avoid double-counting.
       merge_duplicate: Yes, merge them
       keep_both: No, keep both
+      loan_payment: Loan Payment
+      transfer: Transfer
```

YAMLlint (1.37.1)

[error] duplication of key "transfer" in mapping

(key-duplicates)